kubernetes_deploy/development/jobs.yml

```diff
@@ -0,0 +1,20 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  annotations:
+    kubernetes.io/change-cause: "<to be filled in deploy_to_kubernetes script>"
+  name: laa-legal-adviser-api-collect-static
+spec:
+  template:
+    metadata:
+      name: laa-legal-adviser-collect-static
+    spec:
+      restartPolicy: Never
```

Comment:
💭 Do we know what happens when this job fails?
Could we easily find out the reasons for failure by looking at the deployment job?

Reply:
If the Job fails to create, it will fail early

If the Job creates ok, it will spin up a Pod to run it. If the Pod fails, the Job will keep creating new Pods until the timeout (or backoff limit). The reasons for this will be available in `kubectl logs` for any of the Pods created by the Job.

As you suggested offline, I'll add a helpful message to the deploy script output

Reply:
On reflection, I'm going to make these changes in #122, as the handling of Jobs is slightly different there and it seems more straightforward to do it just once